### DISCUSSION TITLE
Scientific Presentations

### DISCUSSION THREAD

**Discussion by ADParedes** [Open-Research-Program/HCOE#10] [Show and tell]
Hi! I have a starting point for Scientific Presentations
Please see the following slide deck and help me to make this more valuable to you all.
[A guide to making a Scientific Presentation Slide Deck: Assertion Evidence Based Slide Deck](https://docs.google.com/presentation/d/1AVTAn6zI7-i5sTeotv6w35y07_1NsLE9/edit#slide=id.p1)

Please comment on this Discussion thread things you would like to learn more about!

**Comment by spulido1** [comment]
Even as covid-19 restrictions ease, online presentations have become increasingly more common. Are there any tips/tricks out there on how to engage an audience during an online presentation? If cameras are off, hard to know if silence means that what I'm saying is being understood or creating confusion.

**Comment by mizquierdo2** [comment]
I would like to learn more about the details regarding the three stylistic differences for research presentation as detailed on slide 22. I have never heard of this before and would like to know when to use one over the other and so forth.

**Comment by jsejo** [comment]
Similar to spulido1, I would like to learn more about how to keep an audience engaged during a virtual presentation. Especially since body movement and stage presence are such big contributors to how the presentation is perceived by the audience. I imagine that these are difficult to convey virtually.

**Comment by bguill3** [comment]
I found it quite interesting that color combination, especially yellow with black, affects the speed at which a reader will take in information. Especially since I rarely see yellow slides in real life. 

I am curious as to whether there is a specific color combination that is preferred/ suggested in general. More specifically for clinical science presentations.
